### ISSUE TITLE
Incompatibility with mail app since 29.4.1

### ISSUE THREAD

**Issue by kriegerse** [GDATASoftwareAG/nextcloud-gdata-antivirus#129]
# Description

After upgrading `nextcloud-gdata-antivirus`  from `29.4.0` to `29.4.1` or `29.4.2` nextcloud stopped working and causing php FATAL error.

# To Reproduce

Possibly:
- Install Nextcloud 
- Install mail App
- Install gdatavaas App

After installation nextcloud stopped working with php fatal error below on double declaration of `Amp\delay()`


# Log

```
www-data@2cc45efbae71:~/html$ ./occ status                                                                                                                                                                                                                    
PHP Fatal error:  Cannot redeclare Amp\delay() (previously declared in /var/www/html/custom_apps/gdatavaas/vendor/amphp/amp/src/functions.php:63) in /var/www/html/custom_apps/mail/vendor/amphp/amp/lib/functions.php on line 133    
```

# Expected behavior

Do not crash nextcloud when installing nextcloud-gdata-antivirus together with mail app

# Versions

- Nextcloud Hub 8 [29.0.6]
- Gdatavaas [29.4.1 / 29.4.2]
- mail [3.7.8]

**Comment by lennartdohmann** [comment]
Hi @kriegerse,

First of all, we would like to apologize for the inconvenience caused and thank you for providing us with the issue. 
I can reproduce the bug and confirm that our app is currently incompatible with the mail app in the versions mentioned. We are on it!

**Comment by lennartdohmann** [comment]
FYI we solved the main problem and have a version compatible with any other dependencies so far. We expect the release this week.

**Comment by lennartdohmann** [comment]
This bug has been fixed as of version 29.4.3

**Comment by kriegerse** [comment]
Thanks for addressing and solving the issue
